pkgs/top-level/all-packages.nix

```diff
@@ -13405,6 +13405,10 @@ in
 
   sundials_3 = callPackage ../development/libraries/sundials/3.x.nix { };
 
+  sundials_lapack = callPackage ../development/libraries/sundials/lapack.nix { };
```

Comment:
We should not add more derivations of sundials, but extend the existing ones with a `lapackSupport` argument (defaulting to true or false, depending on closure size increase)

Reply:
I agree but don't know how to do this.

Reply:
Also mostly you will want lapack support.

Reply:
I meant I don't know how to add a `lapackSupport` argument. I am already using delis.